descartes_core/src/cart_trajectory_pt.cpp

```diff
@@ -203,7 +211,78 @@ bool CartTrajectoryPt::getClosestJointPose(const std::vector<double> &seed_state
                                            const RobotModel &model,
                                            std::vector<double> &joint_pose) const
 {
-  NOT_IMPLEMENTED_ERR(false);
+  Eigen::Affine3d nominal_pose,candidate_pose;
+
+  if(!model.getFK(seed_state,candidate_pose))
+  {
+    return false;
```

Comment:
Should we log an error/warning/info/debug message.  It would be nice to know when the interpolation fails.  Since it should fail rarely, we should probably log something.